src/Playwright/Core/AssertionsBase.cs

```diff
@@ -140,5 +146,8 @@ private string FormatValue(object value)
 
             return value.ToString();
         }
+
+        internal static void SetDefaultTimeout(float timeout)
```

Comment:
Should it be synchronized / atomic to work correctly in multithreaded environment?

Reply:
we don't allow parallelism on the same process aka async/await, it's out of process afaik.